config.json

```diff
@@ -42,6 +42,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 1
+      },
+      {
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "f8afa182-aca3-4464-96b7-9eb7f8d55449",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

Comment:
Do you think that this should be difficulty 2 or 1? (in Python it's 1)

Reply:
Up to you as maintainer. My general rule is that exercises of the same difficulty level should be equivalent in difficulty. `hello-world` would be 1 as the easiest exercise on the track, and `resistor-color` is more difficult than `hello-world` so 2 makes sense to me. On the student-facing side, difficulty levels 1 - 3 are all lumped as "easy" so difficulty 1 or 2 doesn't make a difference to them.

Reply:
I see ;) In my opinion, if leap is level 1, than this also should be, but I'm not going to insist ;) Level 2 is OK for now, there will be probably some cleanup involved right before the release, so if something changes, we can update the difficulty then.